index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Document</title>
+</head>
+  <body>
+   
```

Comment:
I think there shouldn't be emty space for your code seems better looks.

Reply:
> I think there shouldn't be emty space for your code seems better looks.

- [ ] Good one requesting changes politely, however, I would recommend you add the reason you are requesting changes so that the student would not be left in the dark as regards your point of view. Kindly apply this to all your comments.